wordpress-mobile/WordPress-Android
# Issue/fix my site layout on bigger screens

This PR fixes issues with My Site screen layout present on tablets and bigger screens, according to this design:

![proposed-design-1](https://user-images.githubusercontent.com/728822/94510849-a483f680-01cc-11eb-91f1-0e35b5eeba85.png)


What changed:
- Content area of My Site is limited to 500dp width (we eliminated screen size specific padding).
- Quick Action buttons have an equal distance between them withing a 500dp container (they were spaced equally before)

Phone layout:
[![Image from Gyazo](https://i.gyazo.com/6b37503a63e0cf34c3b7bb696f3365a3.png)](https://gyazo.com/6b37503a63e0cf34c3b7bb696f3365a3)

Domain CTA:
[![Image from Gyazo](https://i.gyazo.com/462a0ba0a4a03518230077f4300b8d07.png)](https://gyazo.com/462a0ba0a4a03518230077f4300b8d07)

Tablet:
[![Image from Gyazo](https://i.gyazo.com/514f8acac9a2b79554f7fa2f78e9d1e5.png)](https://gyazo.com/514f8acac9a2b79554f7fa2f78e9d1e5)

To test:
- Make sure My Site layout resembles the proposed design and nothing looks out of place.

PR submission checklist:
